saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -44,7 +45,6 @@ def test_generate_order_payload(
     assert payload.get("shipping_method")
     assert payload.get("shipping_tax_rate")
     assert payload.get("lines")
-    assert payload.get("payments")
```

Comment:
Why do we want to remove it?

Reply:
Code duplication. This assertion is already present a few lines above: https://github.com/mirumee/saleor/blob/d6663a9795ee9df6cfef640d7edb8dc4a217b7ae/saleor/webhook/tests/test_webhook_payloads.py#L44